.flake8

```
@@ -23,3 +23,7 @@ extend-exclude =
                build,
                # nah, don't care
                spack/package.py
+per-file-ignores=
+    python/example/brunel/analysis.py=F405,F403
```

```suggestion
    python/example/brunel/analysis.py:F405,F403
```
and the next two lines.